SF Symbols Link Cell/XXXSymbolsLinkCell.m

```diff
@@ -0,0 +1,154 @@
+#import "XXXSymbolsLinkCell.h"
+
+@implementation UIColor (SymbolsLinkCell)
+
++ (UIColor *)systemColorFromString:(NSString *)stringColor {
```

Comment:
While I don't think this is a very clean idea, this could be reduced significantly with some string manipulation:

```objc
+(UIColor *)systemColorFromString:(NSString *)colorString {
    NSString *finalColorString = [NSString stringWithFormat:@"system%@Color, [colorString capitalizedString]];

    SEL colorSelector = NSSelectorFromString(finalColorString);
    if([UIColor respondsToSelector:colorSelector]) {
        return [UIColor performSelector:colorSelector];
    }

    return nil;
}
```

Also while writing this it looks like Apple has this already implemented in UIColor with `_systemColorWithName:`, taking the full string i.e. systemColorBlack.

Reply:
Love the idea! Gonna fix that.

Reply:
Fixed